fiftyone/core/ontology.py

```
@@ -193,3 +272,99 @@ def from_dict(cls, d: dict) -> "AnnotationOntology":
+    """Checks if an ontology with the given name exists.
+
+    Args:
+        name: the ontology name
+
+    Returns:
+        True/False
+    """
+    return _objects_by_slug(name).count() > 0
+
+
+def delete_ontology(name: str, force: bool = False) -> None:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)